packages/lodestar/package.json

```diff
@@ -82,6 +83,7 @@
     "libp2p-noise": "^1.1.2",
     "libp2p-pubsub": "^0.4.7",
     "libp2p-tcp": "^0.14.6",
+    "lodash": "^4.17.20",
```

Comment:
can you use this package: https://www.npmjs.com/package/lodash.isnil

Reply:
done